VagrantProvisionCentOS7.sh

```diff
@@ -321,6 +324,8 @@ cd ~
 ##### These two are next to do.
 echo "### Installing node-mapnik-server..."
 sudo cp $HOOT_HOME/node-mapnik-server/systemd/node-mapnik.service /etc/systemd/system/node-mapnik.service
+sudo sed -i "s|SERVICE_USER|$VMUSER|g" /etc/systemd/system/node-mapnik.service
+sudo sed -i "s|HOOT_HOME|$HOOT_HOME|g" /etc/systemd/system/node-mapnik.service
```

Comment:
Have the mapnik service file use the correct user and Hootenanny home directory.  Same with export below.

Reply:
Nice!